tests/core/test_utils.py

```diff
@@ -335,7 +335,7 @@ def _assert_error(self, input_arr):
             self.assertEquals(str(excinfo.value), self.error_msg)
 
     def test_malformed_url_raises_an_exception(self):
-        input_arr = ['foo', '/', '/bar', 'baz/', '/fo/o', '/buckets/sbid/scid']
+        input_arr = ['foo', '/', '/bar', 'baz/', '/buckets/sbid/scid']
```

Comment:
Why this change?

Reply:
My understanding is that we need to execute custom code for create, update, delete actions for a  given bucket. This logic is implemented in parse_resource by checking len(parts)==3 hence test input "/fo/o" is not a malformed URL anymore since it satisfies the len(parts) condition in parse_resource. 
Let me know if I am wrong in my understanding of the requirement for triggering custom code around a given bucket or there is a different way you want me to handle this.